Makefile.inc

```diff
@@ -54,6 +54,15 @@ skipclippy:
 
 # Testing. A lot of directories can't test either, yet.
 test:
+	cargo test
 
 skiptest:
 	echo Not doing test for this target
+
+coverage:
+	cargo tarpaulin --out Lcov --output-dir "$(OREBOOT)/coverall"
+	cat `echo "$(OREBOOT)/coverall/lcov.info"`.info >> $(OREBOOT)/coverall/lcov.temps
```

Comment:
This needs to be all the `lcov.info` files in all the `coverall` directories - you can use `find` and `exec` to achieve this